finance-api/overview.md

```diff
@@ -1,18 +1,18 @@
 Finance API overview
-===
-Merchants who take payments with Zettle have an Zettle account that processes transactions for the payments. For example, card payments made with Zettle card readers.
+======
+Merchants who take payments with Zettle have an Zettle account. The account processes transactions for the payments. For example, card payments made with Zettle card readers.
```

Comment:
Not related to this change, but since we went from iZettle to Zettle we should make it `a Zettle account`, not `an Zettle account`.